chapters/ch03.asciidoc

```diff
@@ -825,20 +857,20 @@ const cat = Object.create(baseCat)
 cat.name = 'Milanesita'
 ----
 
-The `Object.create` method is, however, limited to newly created objects. In contrast, we could use +Object.setPrototypeOf+ to change the prototype of an object that already exists, as shown in the following code snippet.
+The `Object.create` method is, however, limited to newly created objects. In contrast, we could use `Object.setPrototypeOf` to change the prototype of an object that already exists, as shown in the following code snippet.
```

Comment:
It occurs to me that using the term "prototype" can be confusing, because it has multiple meanings in a JS context:
1. the `.prototype` property of a constructor function (that will be used as the internal `[[Prototype]]` of the resulting object)
1. The internal `[[Prototype]]` of an object, that is exposed via `Object.getPrototypeOf`/`Object.setPrototypeOf`, that's used for property fallback chain lookups
1. a "first version" of something

If you agree that this can be confusing, consistently disambiguating all uses of the term "prototype" with a qualifier might help?